script.js

```diff
@@ -20,23 +23,29 @@ function checkEnd(penguinPosition) {
     return penguinPosition > 1000
 }
 
-function endGame() {
+function endGameMessage() {
     alert('Far out brah! That was totally radical, you found some sweet lines down those faces! Catch you on the flip')
```

Comment:
it would be good if it would tell you your total time within this end game message (but if there's no time for it no worries)

Reply:
This will be handled in the save score screen and leaderboard as part of story 5/6